misc/python/materialize/checks/mzcompose_actions.py

```diff
@@ -95,7 +95,7 @@ def execute(self, e: Executor) -> None:
             system_settings.add(
                 "ALTER ROLE materialize CREATEROLE CREATEDB CREATECLUSTER;"
             )
-        elif e.current_mz_version >= MzVersion.parse("0.59.0-dev"):
+        elif e.current_mz_version >= MzVersion.parse("0.59.0"):
```

Comment:
This gating disables the check in the current version too I think. So I'm also confused as to why it works. We can submit this for now to get the test green, but I'll have to understand the reason later.